.github/workflows/pages.yml

```diff
@@ -35,7 +35,11 @@ jobs:
           npm run dev &
           sleep 10 &&
           wget --mirror http://localhost:3000 -P out --no-host-directories --page-requisites --adjust-extension &&
-          echo "phillyjs.com" > out/CNAME
+          wget --mirror http://localhost:3000/ad -P out --no-host-directories --page-requisites --adjust-extension
+          echo "phillyjs.com" > out/CNAME &&
+          cp public/favicon.ico out &&
+          cp public/logo.png out &&
```

Comment:
Curiosity: why the `&&`?

Reply:
Entirely laziness; it would probably be nice to format this better!